### ISSUE TITLE
Digest Auth Query Parameter not added to header

### ISSUE THREAD

**Issue by amnesty007** [a85/POSTMan-Chrome-Extension#2299]
Version/App Information:

Postman Version: 4.6.3 / 4.6.2
App (Chrome app or Mac app): Chrome App aswell as Windows App
OS details: Windows 7 64 bit / Windows 10 64 bit
Is the Interceptor on and enabled in the app: off
Did you encounter this recently, or has this bug always been there: recently
Expected behaviour: Correct HTTP Header

When creating a HTTP Digest Auth Request with request parameters in it the request parameters are not added to the URI. Therefor the receiving application is calculating the MD5 Hash over Response and refusing the request with a 401.

Actual URL:
http://localhost:8080/command?TTL=100

This is how it looks:

```
POST http://localhost:8080/command?TTL=100 HTTP/1.1
Content-Type: application/xml
Authorization: Digest username="test", realm="test", nonce="14726152:8ca167409a7", 

**uri=/commands/**,

 response="8f888f159d2fcd32", opaque="4C46D0FD36F6"
Cache-Control: no-cache
Postman-Token: ea7d8739
```

This is how it should look like:

```
POST http://localhost:8080/command?TTL=100 HTTP/1.1
Content-Type: application/xml
Content-Length: xxx
Host: localhost:8080
Connection: Keep-Alive
Authorization: Digest username="test", realm="test", nonce="14726315:a036549ae1a0", 

**uri="/command?TTL=100**, 

response="c0c0f622435191", qop=auth, nc=00000001, cnonce="faebf1a8", opaque="1E9C63"
```

Kind Regards

**Comment by vegetableman** [comment]
@amnesty007 Thanks for reporting this  👍 . The issue has been fixed in our internal build. Will be made part of the next version.

**Comment by vegetableman** [comment]
@amnesty007 The issue has been fixed in the latest version 4.7.2. Check it out.
